### ISSUE TITLE
Research: many-valued applicability states and bifurcation diagnostics

### ISSUE THREAD

**Issue by TeaShaman-cyber** [TeaShaman-cyber/theseus-needle-lab#42]
## Motivation

Historical DeepSeek/Grok research conversations explored two mathematically grounded ideas that may be useful as hypothesis generators for Needle:

1. **Many-valued state semantics** inspired by FDE/Belnap-style values such as `T/B/N/F`, later extended experimentally with an explicit `U = undefined/outside-current-model` state.
2. **Bifurcation / phase-transition framing** for nonlinear regime changes, where a local optimization pressure can cross a critical region and globally alter the system's discrimination behavior.

These discussions were theoretical/emulative. They are **not evidence that the mechanism applies to Needle**. The purpose of this issue is to translate the ideas into falsifiable experiments.

## Candidate mapping to Needle

Current Stage C already separates applicability outcomes such as:

- `NO_CALL`
- `PROBE`
- `READY`
- `UNKNOWN`

A future experiment could treat these not merely as labels but as a **factorized many-valued state** describing distinct epistemic/applicability conditions. One possible research mapping:

- `T` — evidence supports applicability / tool call
- `F` — evidence supports non-applicability / no-call
- `B` — conflicting positive and negative applicability evidence coexist
- `N` — insufficient evidence
- `U` — current learned representation/model cannot classify reliably under its present state/decision geometry

`U` must not become a free-form escape label. It should be derived from measurable conditions and tested for predictive value.

## Bifurcation hypothesis

Stage B exhibited a sharp applicability collapse: positive behavior improved while negative heldout `NO_CALL` collapsed. This suggests testing whether applicability learning undergoes a **regime transition** rather than degrading smoothly.

Candidate observables:

- negative heldout NO_CALL rate
- positive heldout correctness
- dominant semantic share
- entropy / concentration of applicability-state distribution
- conflict mass (`B`-like state)
- unknown/out-of-model mass (`N`/`U`-like state)
- hard-negative sampling pressure
- recovery-buffer occupancy / decay

Research question:

> Is there a reproducible critical region where incremental training pressure causes a discontinuous or hysteretic shift from selective applicability to generalized over-calling?

## Minimal experiments

1. **State-factorization probe**
   - Do not change the trained objective initially.
   - Derive `T/B/N/F[/U]`-style diagnostic states from existing logits/evidence.
   - Test whether these states predict heldout applicability failures better than the current single decision label.

2. **Pressure sweep**
   - Sweep hard-negative sampling / recovery-buffer pressure across preregistered levels.
   - Measure whether heldout behavior changes smoothly or shows a sharp transition.

3. **Hysteresis test**
   - Increase pressure past the suspected transition, then reduce it.
   - Test whether the model returns along the same curve or retains a changed applicability regime.

4. **Early-warning test**
   - Test whether growth of conflict/undefined mass precedes negative-heldout collapse.
   - Acceptance requires predictive value on heldout data, not post-hoc interpretability.

## Guardrails

- Historical conversations are hypothesis provenance only, not scientific evidence.
- No fixed “15% threshold” is assumed; any critical point must be measured from Needle experiments.
- Heldout remains evaluation-only.
- No free-form rationale / CoT state.
- Canonical diagnostic state should be structured JSON with bounded enums.
- This issue must not modify or delay the preregistered Stage C run unless a separate decision explicitly promotes one of these hypotheses.

## Why this may matter

If applicability collapse is a regime transition rather than ordinary scalar degradation, then optimizing only average loss/accuracy can miss the approach to failure. A many-valued diagnostic state plus bifurcation-style sweeps could provide earlier warning and a better control variable for future recovery curricula.

**Comment by TeaShaman-cyber** [comment]
### Tech review 2026-09-08 — reasoning dynamics reference

Primary source: https://arxiv.org/abs/2609.04963 — **Fractal basins trap latent reasoning**.

The paper gives this issue a useful dynamical-systems reference: in recurrent/looped reasoning models, harder tasks show transient-chaotic trajectories and long dwell times near saddle-like, nearly-correct states. That is conceptually adjacent to this issue's bifurcation/regime-transition hypothesis, but it is **not evidence that Needle currently has the same latent geometry**.

Useful bounded extension to the research note:

```text
regime transition diagnostics
        +
possible stall / near-solution diagnostics
```

Only promote this into a Needle experiment if an observable state/trajectory proxy can be defined without inventing hidden-state access. Candidate measurable proxies should come from existing logits, margins, repeated routing states, or checkpoint-to-checkpoint behavior; no free-form CoT state.

Related new review line: #47 explores state-dependent local credit gating; #42 remains the right home for the broader dynamical-regime question.

Epistemic update: `PRIMARY SOURCE VERIFIED / TRANSFER TO NEEDLE UNPROVEN / NO EXPERIMENT AUTHORIZED BY THIS COMMENT`
